zeton/templates/child_info.html

```diff
@@ -80,43 +79,52 @@ <h4>NAGRODY</h4>
     <div id="bany_warny" class="square">
```

Comment:
w klasie tego diva brakuje `col-xl-6` przez co przy zmianie wielkości okna 'odstaje' od reszty, ale to kosmetyka. Może się czepiam :)

Reply:
poprawiłem

Reply:
Gdyby  były jakieś problemy z wyświetlaniem to proszę wyczyścić dane przeglądarki.